### DISCUSSION TITLE
My comment on any post should be the first one among all.

### DISCUSSION THREAD

**Discussion by ashishk1331** [dailydotdev/daily#903] [Feature Requests]
It would be nice if my **comment** on any post **showed up first** rather than at the very bottom of the list.
This tells that I'd already commented on the post and I don't need to scroll to the very end in order to look what I'd written.

![Screenshot_20230523-004602_Brave.jpg](https://github.com/dailydotdev/daily/assets/44055846/10e138f3-7852-45c4-972b-9c818e3648ab)

**Comment by idoshamun** [comment]
Hi @ashishk1331,

Once you comment on a post, the comment icon changes its color, like the image below:
<img width="655" alt="image" src="https://github.com/dailydotdev/daily/assets/1993245/b3500db3-8e8a-48e2-a3f6-604ca1066faa">

Does that make sense?

**Reply by ashishk1331** [reply]
Would clicking on the comment icon move me to my comment?

**Reply by idoshamun** [reply]
No, you will be able to add one more comment

**Reply by ashishk1331** [reply]
Now that functionality is great!

However, what I am trying to say is that once I'd posted my comment it should display at the top of all comments, I.e., the first comment in all comments of that post. 

Similar to Twitter, if you add a comment to a tweet, your comment will be the first one to be displayed in the comments of that tweet no matter how many times you visit that post.

Am I not able to explain it clear? Please tell me if that's the case.

**Reply by idoshamun** [reply]
You're all clear. 100% understand your case :)
